### DISCUSSION TITLE
Change IP

### DISCUSSION THREAD

**Discussion by rzivny** [intuitem/ciso-assistant-community#22] [Q&A] [answered]
I have a request, how do I change (127.0.0.1 localhost) to an IP address

**Comment by eric-intuitem** [comment]
Hi rzivny, you can use the CISO_URL environment variable, as explained in the readme.

**Reply by rzivny** [reply]
Create a file in the parent folder (e.g. ../myvars) and store your environment variables within it by copying and modifying the following code and replace "" by your private values. Take car not to commit this file in your git repo.?? Where should I put this file please?

**Reply by rzivny** [reply]
in ciso-assistant-community i create folder myvars. what file should i create how to map it? thanks a lot

**Reply by rzivny** [reply]
I admit that I don't understand the procedure

**Reply by eric-intuitem** [reply]
Hi rzivny, you can put the env file where you want and name it as you want. It is not a folder, only a single file. Once you have created it, just ingest it by typing “source myvars” (supposing you called it myvars). Then the environment variables will be loaded in your environment, you can check that with the “env” command. You are then ready to start CISO Assistant with your new parameters.

**Reply by rzivny** [reply]
thanks, must be a *.py extension for myvars?

**Reply by eric-intuitem** [reply]
It would not make sense, at it is not a python file. Better add no extension.

**Reply by rzivny** [reply]
I did it as you wrote but it doesn't work, it still loads http://127.0.0.1

**Reply by rzivny** [reply]
I load via env, I see a new IP but when I give the command python manage.py runserver the adress is 127.0.0.1

**Comment by eric-intuitem** [comment, marked as answer]
Hi rzivny, indeed this is not sufficient. I have pushed a fix for the name of the environment variable CISO_URL, so first update your repo. Then you can type, assuming you want to server 10.1.2.3 on port 8000:
CISO_URL=http://10.1.2.3:8000 python manage.py runserver 10.1.2.3:8000
Obviously this is for test only, a production environment should use https.

**Reply by ALFAT33** [reply]
BAGAIMANA MERUBAH ALAMAT IP . HINGA MASUK KE SITUS  YANG MEMILIK IP PRIBADI 
![image](https://github.com/intuitem/ciso-assistant-community/assets/154671927/c5c832f1-aa46-466b-a18e-061819a9083b)

**Comment by rzivny** [comment]
Hi Eric, excellent, I confirm - it works without problems. Thanks a lot.

**Comment by ALFAT33** [comment]
BAGAIMANA MERUBAH ALAMAT IP . HINGA MASUK KE SITUS  YANG MEMILIK IP PRIBADI 
![image](https://github.com/intuitem/ciso-assistant-community/assets/154671927/c5c832f1-aa46-466b-a18e-061819a9083b)

**Comment by eric-intuitem** [comment]
Hi @ALFAT33, with CISO Assistant 0.9.6 you have to set:
- in the backend: the environment variable CISO_ASSISTANT_URL to where your frontend or proxy is (typically http://localhost:5173) 
- in the frontend:
     - the environment variable PUBLIC_BACKEND_API_URL to where your backend is (typically http://localhost:8000/api)
     - if you have a proxy, the environment variable ORIGIN to where your frontend is or proxy is (e.g.  https://localhost:8443)
- Please have a look at the docker-compose.yaml file, it provides a working configuration that can be used as an example.
